md-dao/src/main/java/com/wipro/ats/bdre/md/dao/HistoryDataImportDAO.java

```diff
@@ -505,12 +499,12 @@
 
                     PropertiesId rawTableNamePropertiesId = new PropertiesId();
                     rawTableNamePropertiesId.setProcessId(file2Raw.getProcessId());
-                    rawTableNamePropertiesId.setPropKey("table_name");
+                    rawTableNamePropertiesId.setPropKey("TABLE_NAME");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "TABLE_NAME" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)